### PR TITLE
Re-enable safety checking

### PR DESCRIPTION
**Pull request by K-Ho**
## Description
Re enables safety checking.
## Questions
- 
-
-

### Fixes
- Fixes # [Link to Issue]

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](./CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: core/vm/evm.go [hunk 1]
```diff
@@ -71,11 +71,10 @@ func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, err
...
 
-		// Temporary: Safety checker always returns true.
-		// ovmTODO: Remove this.
-		if contract.Address() == OvmStateDump.Accounts["OVM_SafetyChecker"].Address {
-			return AbiBytesTrue, nil
-		}
+		// Uncomment to make Safety checker always returns true.
+		// if contract.Address() == OvmStateDump.Accounts["OVM_SafetyChecker"].Address {
+		// 	return AbiBytesTrue, nil
+		// }
```
Comment (karlfloersch) [inline core/vm/evm.go:77]: Have you tested this @K-Ho I presume?

**Review by karlfloersch — APPROVED** [review]
This LGTM assuming you tried this out

File: core/vm/evm.go (hunk 1 above)
Comment (K-Ho) [reply core/vm/evm.go:77]: yep!

Comment (K-Ho) [reply core/vm/evm.go:77]: tested a full synthetix deploy with safety checking enabled

**Pull request merged**